src/modules/auth/controllers/auth.controller.ts

```diff
@@ -15,29 +18,28 @@ export class AuthController {
 
   @Post('/signup')
   public async signUp(
-    @Body() userDto: Pick<UserDTO, 'email' | 'password'>,
+    @Body() userDto: CreateUserResponseDto,
   ): Promise<boolean> {
     return this.authService.singUp(userDto);
   }
 
-  @UseGuards(RegisterGuard)
-  @Get('/email/confirm')
+  @Get('/email/confirm/:token')
   public async verifyEmail(@Param('token') token: string): Promise<boolean> {
     return this.authService.verifyEmail(token);
   }
 
   @UseGuards(RegisterGuard)
   @Post('/login')
   public async signIn(
-    @Body() userDto: Pick<UserDTO, 'email' | 'password'>,
+    @Body() userDto: CreateUserResponseDto,
   ): Promise<UserEntity> {
```

Comment:
you should create `UserResDto` as a response type instead of entity with fields you want to return

Reply:
in future it would be used in swagger documentation